src/microservices/Microservices.CohortPackager/Execution/ExtractJobStorage/MongoDB/MongoExtractJobInfoExtensions.cs

```diff
@@ -17,5 +17,19 @@ public static ExtractJobInfo ToExtractJobInfo(this MongoExtractJobDoc mongoExtra
                 mongoExtractJobDoc.IsIdentifiableExtraction,
                 mongoExtractJobDoc.IsNoFilterExtraction
                 );
+
+        public static CompletedExtractJobInfo ToExtractJobInfo(this MongoCompletedExtractJobDoc mongoCompletedExtractJobDoc)
```

Comment:
I think I structured it this way so that we could potentially have multiple `ExtractJobStore` implementations that each have their own internal representation of the job state, but the `ExtractJobInfo` class would be the common data class between them and the reporting layer. The `Mongo...Doc`s are what actually get serialized, and so I also didn't want to couple that to any sort of inheritance.

Reply:
Ok if you are happy with it its not a big issue.